testnets/sgenetworktestnet/assetlist.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_id": "sgenetworktestnet",
+  "assets": [
+    {
+      "description": "The SGE token is primarily a governance token for the SGE Network blockchain.",
+      "denom_units": [
+        {
+          "denom": "usge",
+          "exponent": 0,
+          "aliases": []
+        },
+        {
+          "denom": "usge",
+          "exponent": 6,
+          "aliases": []
+        }
+      ],
+      "base": "usge",
+      "name": "SGE Network",
+      "display": "sge",
+      "symbol": "SGE",
+      "logo_URIs": {
+        "png": "./images/logo.png",
```

Comment:
please reference the full url to this repo, e.g., "https://raw.githubusercontent.com/cosmos/chain-registry/master/sgenetworktestnet/images/logo.png"